src/Access/UsersConfigAccessStorage.cpp

```diff
@@ -65,13 +65,16 @@ namespace
         bool has_ldap = config.has(user_config + ".ldap");
         bool has_kerberos = config.has(user_config + ".kerberos");
 
-        size_t num_password_fields = has_no_password + has_password_plaintext + has_password_sha256_hex + has_password_double_sha1_hex + has_ldap + has_kerberos;
+        const auto certificates_config = user_config + ".certificates";
+        bool has_certificates = config.has(certificates_config);
+
+        size_t num_password_fields = has_no_password + has_password_plaintext + has_password_sha256_hex + has_password_double_sha1_hex + has_ldap + has_kerberos + has_certificates;
         if (num_password_fields > 1)
-            throw Exception("More than one field of 'password', 'password_sha256_hex', 'password_double_sha1_hex', 'no_password', 'ldap', 'kerberos' are used to specify password for user " + user_name + ". Must be only one of them.",
+            throw Exception("More than one field of 'password', 'password_sha256_hex', 'password_double_sha1_hex', 'no_password', 'ldap', 'kerberos', 'certificates' are used to specify password for user " + user_name + ". Must be only one of them.",
```

Comment:
what about changing 'password' to 'authentication info' or somehting like this?
my concern is about the log text in general - this phrase may be already part of some log-scraping systems that possibly monitor ClickHouse logs and have a bunch of regex rules base on the original phrase. changing it would possibly break those systems/rules in a very subtle and hard-to-detect way
while 'password' doesn't sound right (though still kind of acceptable) with the certificates, maybe avoiding the above mentioned risk is more important?

Reply:
Let's change to `authentication info`. About your concern about log scrapping systems, we change log messages sometimes and I believe it has never been promised anywhere to keep the log messages in some specific form, so I think we can change log messages.

Reply:
ok